docs/REST API/rest-api.md

```diff
@@ -9532,6 +9532,8 @@ Add a package (.pkg, .msi, .exe, .deb, .rpm) to install on macOS, Windows, or Li
 | pre_install_query  | string | form | Query that is pre-install condition. If the query doesn't return any result, Fleet won't proceed to install. |
 | post_install_script | string | form | The contents of the script to run after install. If the specified script fails (exit code non-zero) software install will be marked as failed and rolled back. |
 | self_service | boolean | form | Self-service software is optional and can be installed by the end user. |
+| automatic_install | boolean | form | Automatically create policy that triggers install if software isn't installed on the host. |
```

Comment:
@marko-lisica We should document that we are returning HTTP 424 when the installer was created successfully but not the policy. See comment from @georgekarrv [here](https://github.com/fleetdm/fleet/issues/24385#issue-2718676033).

Reply:
@lucasmrod Can we return error message explaining this if it happens. See error message in UI [here](https://www.figma.com/design/yGJoxOTdycccBpatm1KTfe/%2323344-Create-policies-automatically-for-custom-packages?node-id=5402-3299&node-type=instance&t=LHAFtLgx1hZyWQzT-11).

Reply:
I will attempt to do this operation atomically (either both the installer and policy are created or none, will let you know how that goes).